codeforboston/maple
# Migrating Tabs component to Bootstrap

# Summary

This PR migrates the Tabs component to Bootstrap - See #1456 

# Checklist

- [na] On the frontend, I've made my strings translate-able.
- [na] If I've added shared components, I've added a storybook story.

# Screenshots

N/A (No visible change)

# Known issues

N/A

# Steps to test/reproduce

1. Go to the Learn Testimony page
1. Click between the tabs and verify that the tabs nav looks as expected
